digital_image_processing/image_watersheding_custom_markers.py

```diff
@@ -0,0 +1,89 @@
+import cv2
+import numpy as np
+from matplotlib import cm
+
+# loading the image (kept in the same directory)
+img = cv2.imread('/image_data/scenery.jpg')
+img_copy = img.copy()
+
+# defining marker image & segmented image
+marker_image = np.zeros(img.shape[:2], dtype=np.int32)
+segments = np.zeros(img.shape, dtype=np.uint8)
+
+"""
+creating function for defining color
+"""
+
+
+def create_rgb(i):
```

Comment:
Please avoid single-letter variable names especially for function parameters.  They make your code look old fashioned.  Please use a self-documenting name and add Python type hints for function parameters and return types.